src/parsers/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright 2012-2019 CNRS-UM LIRMM, CNRS-AIST JRL
+#
+
+find_package(tinyxml2 REQUIRED)
```

Comment:
This causes the CI to fail on Linux as the tinyxml2 version shipped by Ubuntu does not have a proper CMake script

Reply:
Oops, don't know why this line reappeared since @arntanguy made a patch for this. I probably messed up my rebase. I'll remove it